libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Actions/Microsoft.TraceActivity.schema

```diff
@@ -1,8 +1,8 @@
 {
     "$schema": "https://raw.githubusercontent.com/microsoft/botbuilder-dotnet/4.Future/schemas/component.schema",
     "$role": "unionType(Microsoft.IDialog)",
-    "title": "Trace Activity Action",
-    "description": "This is a action which sends an TraceActivity to the transcript",
+    "title": "Send a TraceActivity",
+    "description": "Send a trace activity",
```

Comment:
send a trace activity to the transcript and/or emulator.

Reply:
Updated.